src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -177,10 +178,10 @@ private void CheckPackageVulnerabilities(IReadOnlyList<IReadOnlyDictionary<strin
                     foreach (var advisory in auditInfo.GraphsPerVulnerability.Keys)
                     {
                         int severity = advisory.Severity;
-                        if (severity == 0) { Sev0DirectMatches++; }
-                        else if (severity == 1) { Sev1DirectMatches++; }
-                        else if (severity == 2) { Sev2DirectMatches++; }
-                        else if (severity == 3) { Sev3DirectMatches++; }
+                        if (severity == (int)PackageVulnerabilitySeverity.Low) { Sev0DirectMatches++; }
+                        else if (severity == (int)PackageVulnerabilitySeverity.Moderate) { Sev1DirectMatches++; }
+                        else if (severity == (int)PackageVulnerabilitySeverity.High) { Sev2DirectMatches++; }
+                        else if (severity == (int)PackageVulnerabilitySeverity.Critical) { Sev3DirectMatches++; }
```

Comment:
I see almost identical code in the same class file lines 195 to 198. Can we extract these lines to a separate method and invoke in these places it to avoid duplication? I am okay with creating a follow-up PR to address this comment.